src/main/java/com/direwolf20/buildinggadgets/client/gui/ModeRadialMenu.java

```diff
@@ -196,7 +205,7 @@ public void drawScreen(int mx, int my, float partialTicks) {
 
         if (!tool.isEmpty()) {
             GlStateManager.scale(s, s, s);
-            GlStateManager.translate(x / s - 8, y / s - 8, 0);
+            GlStateManager.translate(x / s - offset, y / s - 8, 0);
```

Comment:
Is it intended to apply the offset to only one of the coordinates? (offset is 8.5, right? so we translate here a bit less x than y...)

Reply:
It's because the textures of some of the gadgets are not centered in the x-axis, as a result of being only 16x16 resolution.